kubernetes/docker-default-networkpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  creationTimestamp: null
+  name: docker-default
+spec:
+  ingress:
+    - from:
+        - podSelector:
+            matchLabels:
+              io.kompose.network/docker-default: "true"
+  podSelector:
+    matchLabels:
+      io.kompose.network/docker-default: "true"
```

Comment:
This seems to be a new `kompose` addition and is unrelated to my simplification.